conanfile.py

```diff
@@ -0,0 +1,25 @@
+from conans import ConanFile
+from conans.tools import download, untargz
+import os
+
+class TypeSafe(ConanFile):
+    name = 'type_safe'
+    url  = 'https://foonathan.github.io/blog/2016/10/11/type-safe.html'
```

Comment:
The URL should point to the repo containing the recipe. It is the entry point to collaborate and give feedback.

Reply:
@Manu343726 Could you fix it?